operation/fhir-operation-erase/src/main/java/com/ibm/fhir/operation/erase/util/EraseParametersBuilder.java

```diff
@@ -0,0 +1,228 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.fhir.operation.erase.util;
+
+import static com.ibm.fhir.model.type.String.string;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import com.ibm.fhir.model.resource.Parameters;
+import com.ibm.fhir.model.resource.Parameters.Parameter;
+import com.ibm.fhir.model.type.Code;
+import com.ibm.fhir.model.type.Extension;
+
+import net.jcip.annotations.ThreadSafe;
+
+/**
+ * Supports various use cases where a parameters object must be generated
```

Comment:
When do we need to generate a Parameters resource?
If its about converting query params into a parameters object, I think our operation framework already does this.

Reply:
Where is it used?

Reply:
Yes - it does, this is a helper method for JavaCode/Consumer who want to assemble a valid InputParameter. This is used in the Test Code - IT/UT

Reply:
OK, I'm wondering if it should be moved to `src/test/java` if its not used by the operation impl.  Fine with whatever you decide.

Reply:
Moving it is no problemo...